portfolio/src/main/java/com/google/sps/servlets/AddGetCommentsServlet.java

```diff
@@ -30,8 +30,8 @@
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that creates and lists comments data. */
-@WebServlet("/data")
-public class DataServlet extends HttpServlet {    
+@WebServlet("/add-get-comments")
+public class AddGetCommentsServlet extends HttpServlet {    
```

Comment:
CommentsServlet would be a better name.
We don't usually use words like "add" and "get" in the class name. They're just for the function names.

Reply:
Got it. Done